test/helpers/index.js

```diff
@@ -10,8 +12,18 @@ module.exports = {
   bbKeys: {
     generate () {
       const keys = ssbKeys.generate()
-      keys.id = keys.id.replace('ed25519', 'bbfeed-v1')
-      return keys
+
+      return {
+        public: prune(keys.public),
+        secret: prune(keys.private),
```

Comment:
I think it's best to leave these as is. They key is still ed25519. ssb-keys can handle both I just think it is easier to read with the .ed25519 extension.